src/App.tsx

```diff
@@ -1,73 +1,107 @@
-import React from 'react';
+import React, { useCallback, useMemo, useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Autocomplete } from './components/Autocomplete';
+import { Person } from './types/Person';
+
+// eslint-disable-next-line @typescript-eslint/ban-types
+const debounce = (callback: Function, delay: number = 300) => {
```

Comment:
The debounce function should allow for a customizable delay via props. Consider passing the delay as a parameter to the `debounce` function to make it configurable.